.changeset/lemon-ladybugs-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": patch
+---
+
+Add kv store tied to jobs and use it for juels fee per coin cache to store persisted values for backup
```

Comment:
Can we phrase this in a way that node ops will understand? e.g. by referencing the relevant job spec field

Reply:
Job spec changes are already refferenced in previous changeset when the cache was added

Reply:
Do I  even need to add changesets that aren't useful for NOPs?

Reply:
We now almost always require changeset because of new CI